docs/src/main/asciidoc/security-oidc-bearer-token-authentication-tutorial.adoc

```diff
@@ -256,15 +255,16 @@ For more information, see the xref:security-keycloak-admin-client.adoc[Quarkus K
 include::{includes}/devtools/dev.adoc[]
 ====
 * xref:security-openid-connect-dev-services.adoc[Dev Services for Keycloak] will start a Keycloak container and import a `quarkus-realm.json`.
-. Open a xref:dev-ui.adoc[Dev UI], which you can find at http://localhost:8080/q/dev-ui[/q/dev-ui], then click a `Provider: Keycloak` link in an `OpenID Connect` `Dev UI` card.
+. Open a xref:dev-ui.adoc[Dev UI], which you can find at http://localhost:8080/q/dev-ui[/q/dev-ui].
+Then, in an OpenID Connect `Dev UI` card, click the `Keycloak provider` link .
```

Comment:
```suggestion
Then, in an `OpenID Connect` card, click the `Keycloak provider` link .
```

Suggesting this minor update because Dev UI contains the cards such as `OpenId Connect`